using-mobile-extensions/adobe-campaign-standard/adobe-campaign-standard-api-reference.md

```diff
@@ -226,3 +273,188 @@ ACPCore.setPushIdentifier("pushIdentifier");
 {% endtab %}
 {% endtabs %}
 
+## Tracking push or local notification message interactions
+
+User interactions with local or push notifications can be tracked by invoking the `ACPCore::CollectMessageInfo` API. After the API is invoked, a network request will be made to Campaign containing the message interaction event. For more info on Campaign message interaction events, see [Implementing local notification tracking](https://helpx.adobe.com/campaign/kb/local-notification-tracking.html#Description). 
```

Comment:
Please reword the 2nd and 3rd sentences as follows:
"After the API is invoked, a network request is made to Campaign that contains the message interaction event. For more information about Campaign message interaction events, see [Implementing local notification tracking](https://helpx.adobe.com/campaign/kb/local-notification-tracking.html#Description). "